test/ComposerRequireCheckerTest/FileLocator/LocateComposerPackageSourceFilesTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace ComposerRequireCheckerTest\FileLocator;
+
+use ComposerRequireChecker\FileLocator\LocateComposerPackageSourceFiles;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @todo We're missing tests for the "classmap" and "files" keys.
+ */
+class LocateComposerPackageSourceFilesTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\FileLocator\LocateComposerPackageSourceFiles */` to prevent coverage leaks